indico/migrations/versions/20210627_2317_420195768776_add_reminder_send_to_speakers_column.py

```diff
@@ -0,0 +1,27 @@
+"""Add reminder send_to_speakers column
+
+Revision ID: 420195768776
+Revises: 90384b9b3d22
```

Comment:
Can you rebase and update the revision since we added other DB changes in `master` in the meantime?

- Revises / down_revision needs to be `5cbb0eb12eb3`
- The file name needs to have a date stamp greater than `20210630_1349` for correct sorting

Reply:
No problem. I force-pushed the rebased version w/ the fixes to your branch since it was a minor change anyway and I needed it fixed in order to test it.